lib/pig_latin_cases.rb

```diff
@@ -1,5 +1,3 @@
-require 'exercise_cases'
```

Comment:
Not entirely sure I agree with removing this line.
Sure `require_all` is awesome, but there's only one dependency and it's nice to be able to see where `ExerciseCase` comes from.